recipes/app1.rb

```diff
@@ -137,3 +137,11 @@
     '8082 -c config/unicorn.rb -E deployment -D'
   end
 end
+
+# Logrotate
+logrotate_app 'OpenID' do
+  path '/home/openid-production/shared/log/*'
+  options %w(copytruncate)
```

Comment:
It's an upgrade over copytruncate. Copytruncate can potentially lose a sliver of data during the rotation.